packages/Manager/sample-data/observation_data_function.sql

```diff
@@ -0,0 +1,741 @@
+            (p_observation_measurement_id, '2022-12-13 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-14 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-14 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-15 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-15 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-16 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-16 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-17 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-17 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-18 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-18 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-19 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-19 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-20 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-20 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-21 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-21 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-22 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-22 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-23 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-23 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-24 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-24 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-25 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-25 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-26 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-26 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-27 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-27 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-28 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-28 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-29 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-29 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-30 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-30 23:50:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-31 00:00:00+00', 3475250),
+            (p_observation_measurement_id, '2022-12-31 23:50:00+00', 3475250);
+END;
+$$ LANGUAGE plpgsql;
```

Comment:
Seems a little unusual to me that this seed data is embedded in a function. But I guess without more dedicated scripting to manage the whole process, not sure where else it could go for now.